requirements/requirements.in

```diff
@@ -29,7 +29,7 @@ djangorestframework>=3.12.1
 djangorestframework-yaml
 GitPython>=3.1.1  # minimum to fix https://github.com/ansible/awx/issues/6119
 irc
-jinja2>=2.11.0  # required for ChainableUndefined
+jinja2>=2.11.3  # CVE-2020-28493
```

Comment:
Can this be pinned in `requirements.txt`? I worry that it will be easy to override here.